package.json

```diff
@@ -40,6 +40,7 @@
     "lodash": "^3.10.1",
     "mocha": "^2.2.5",
     "pivotaltracker": "^0.1.4",
+    "pug": "^2.0.0-alpha1",
```

Comment:
I saw that - willing to give it a go - all dependent packages re pinned to a Git tag, if all hell breaks loose, will just revert to the previous pin and wait for updates